cddagl/resources/launcher.ico.license

```diff
@@ -1,5 +1,5 @@
-SPDX-FileCopyrightText: 2021 Paul Davey aka Mattahan <https://mattahan.deviantart.com/>
+SPDX-FileCopyrightText: 2022 Aunt Su <https://github.com/I-am-Erk/CDDA-Tilesets/>
 
 SPDX-License-Identifier: CC-BY-NC-SA-2.5
```

Comment:
the sprite is CC BY SA, not NC

Reply:
Thanks for the heads-up! Will rectify in a followup commit 👍🏼